control/content/app.services.js

```diff
@@ -428,27 +427,27 @@
       }
 
       return {
-        initStateSeeder: function() {
-          stateSeederInstance = new buildfire.components.aiStateSeeder({
-            generateOptions: {
-              userMessage: `Generate a sample of redeemable items for a new [business-type].`,
-              maxRecords: 5,
-              systemMessage:
-              'listImage URL related to title and the list type. use source.unsplash.com for image URL, URL should not have premium_photo or source.unsplash.com/random, cost to redeem which is a number greater than zero and less than 100, return description as HTML.',
-              jsonTemplate: generateJSONTemplate,
-              callback: handleAIReq.bind(this, false),
-              hintText: 'Replace values between brackets to match your requirements.',
-            },
-            importOptions: {
-              jsonTemplate: importJSONTemplate,
-              sampleCSV: "Hotel Voucher, 50, 10, Redeem this voucher for a one-night stay at a luxurious hotel of your choice, https://source.unsplash.com/featured/?hotel\nFlight Upgrade, 30, 15, Upgrade your economy class ticket to business class, https://source.unsplash.com/featured/?flight\nCity Tour, 20, 5, Explore the city with a guided tour that covers all the major attractions and landmarks, https://source.unsplash.com/featured/?city\nAdventure Activity, 80, 30, Embark on an adrenaline-pumping adventure activity such as bungee jumping or skydiving, https://source.unsplash.com/featured/?adventure",
-              maxRecords: 15,
-              hintText: 'Each row should start with a loyalty item title, cost to redeem, points earned, description, and image URL.',
-              systemMessage: ``,
-              callback: handleAIReq.bind(this, true),
-            },
-          }).smartShowEmptyState();
-        },
+      initStateSeeder: function() {
+  stateSeederInstance = new buildfire.components.aiStateSeeder({
+    generateOptions: {
+      userMessage: `Generate a sample of redeemable items for a new [business-type].`,
+      maxRecords: 5,
+      systemMessage:
+      'listImage URL related to title and the list type. Use https://app.buildfire.com/api/stockImages/?topic=title&imageType=medium , A maximum of 2 comma-separated title can be used for each URL.  cost to redeem which is a number greater than zero and less than 100, return description as HTML.',
```

Comment:
listImage URL related to the title or the list type. Use https://app.buildfire.com/api/stockImages/?topic={topic}&imageType=medium, A maximum of 2 comma-separated topics can be used for each URL. cost to redeem which is a number greater than zero and less than 100, return description as HTML.